portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello world!', 'I have 2 cats :3!', '你好，世界！', 'Bonjour le monde!', 'True power of use human beings is that we can change ourselves on our own- Saitama.', 'You don\'t need to be on anyone\'s side but your own.- Itaewon Class'];
```

Comment:
This is a really long line; consider placing each element of the list on its own line.